generators/app/index.js

```diff
@@ -95,6 +97,31 @@ module.exports = generators.Base.extend({
       }.bind(this));
     },
 
+    askForGithubAccount: function () {
+      if (this.pkg.repository) {
+        this.props.githubAccount = this.pkg.repository.split('/')[0];
+        return;
+      }
+
+      var done = this.async();
+
+      askGithubAccount({
```

Comment:
This does not call for the complexity of moving that logic out of the generator.

I did this with the module name question because composition will require other generators to ask the same question, and having a reusable method is a must. But it is still dirty, so let's not do it if it's not required.